src/Symfony/Bundle/SecurityBundle/Tests/Functional/UserPasswordEncoderCommandTest.php

```diff
@@ -86,7 +86,7 @@ public function testEncodePasswordArgon2i()
         $this->assertContains('Password encoding succeeded', $output);
 
         $encoder = new Argon2iPasswordEncoder();
-        preg_match('#  Encoded password\s+(\$argon2id\$[\w\d,=\$+\/]+={0,2})\s+#', $output, $matches);
+        preg_match('#  Encoded password\s+(\$argon2i\$[\w\d,=\$+\/]+={0,2})\s+#', $output, $matches);
```

Comment:
Wouldn't `#  Encoded password\s+(\$argon2id?\$[\w\d,=\$+\/]+={0,2})\s+#` work then?

Reply:
Thank you for fixing it. :+1: